### ISSUE TITLE
Missing file totalwithmaininfo.csv

### ISSUE THREAD

**Issue by ptdung** [sandyying/APM-Drowsiness-Detection#1]
Hi there.
I am trying your project and get this issue. 
Can you provide me the csv file?

**Comment by youjin-c** [comment]
I also find it hard to follow on making totalwithmaininfo.csv. If you cannot provide the file, could you tell me the csv file structure?

**Comment by 00susant** [comment]
same here, Its hard to follow making totalwithmaininfo.csv

**Comment by kcLin2708** [comment]
Can you please provide this csv file? Thank you!

**Comment by kcLin2708** [comment]
Can you please provide us this csv file? Thank you.

**Comment by harsbub** [comment]
where can i find this csv file?

**Comment by Anim0011** [comment]
Please provide correct code.

**Comment by hackenjoe** [comment]
Seems like that the repo is incomplete. Has someone figured out what the csv file could be?

**Comment by hackenjoe** [comment]
You can check my repo, where I also added this file.
https://github.com/hackenjoe/drowsiness_detection

**Comment by NaikKhushi21** [comment]
in the normalization code, while creating mean_EAR columns in the dataframe, I am getting an error stating KeyError: 20. Can anyone pls help me solve that?

**Comment by NaikKhushi21** [comment]
Here

![Screenshot (82)](https://user-images.githubusercontent.com/80700736/216770029-56d49cd5-e001-4085-917a-44e6d12c3bc5.png)

**Comment by NaikKhushi21** [comment]
Here

![Screenshot (82)](https://user-images.githubusercontent.com/80700736/216770098-a3ef02d3-d771-4c22-bed3-558033665076.png)
